timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/AutobatchingPaxosLearnerNetworkClientFactory.java

```diff
@@ -119,28 +133,10 @@ public void learn(long seq, PaxosValue value) {
                 }
                 throw new RuntimeException(cause);
             } catch (InterruptedException e) {
-                // TODO(fdesouza): handle
+                Thread.currentThread().interrupt();
```

Comment:
(Discussed elsewhere - at some point this should probably be unified somewhere, but for now this is fine)